extractor/filesystem/list/list.go

```diff
@@ -176,19 +177,20 @@ var (
 
 	// OS extractors.
 	OS = InitMap{
-		dpkg.Name:     {dpkg.NewDefault},
-		apk.Name:      {apk.NewDefault},
-		rpm.Name:      {rpm.NewDefault},
-		cos.Name:      {cos.NewDefault},
-		snap.Name:     {snap.NewDefault},
-		nix.Name:      {nix.New},
-		module.Name:   {module.NewDefault},
-		vmlinuz.Name:  {vmlinuz.NewDefault},
-		pacman.Name:   {pacman.NewDefault},
-		portage.Name:  {portage.NewDefault},
-		flatpak.Name:  {flatpak.NewDefault},
-		homebrew.Name: {homebrew.New},
-		macapps.Name:  {macapps.NewDefault},
+		dpkg.Name:             {dpkg.NewDefault},
+		apk.Name:              {apk.NewDefault},
+		rpm.Name:              {rpm.NewDefault},
+		cos.Name:              {cos.NewDefault},
+		snap.Name:             {snap.NewDefault},
+		nix.Name:              {nix.New},
+		module.Name:           {module.NewDefault},
+		vmlinuz.Name:          {vmlinuz.NewDefault},
+		pacman.Name:           {pacman.NewDefault},
+		portage.Name:          {portage.NewDefault},
+		flatpak.Name:          {flatpak.NewDefault},
+		homebrew.Name:         {homebrew.New},
+		macapps.Name:          {macapps.NewDefault},
+		vscodeextensions.Name: {vscodeextensions.New},
```

Comment:
This is not really a OS package extractor. Might be better to create a new group called "Misc" and move it there. While you're at it you could also move the Wordpress extractor from the Language list to Misc.